### PR TITLE
Updates and fixes for Visibility and Range

### PR DESCRIPTION
**Pull request by mfunk**

### CODE REVIEW

**Comment by mfunk** [comment]
@csmoore can you check this one out?

File: visibility/toolboxes/scripts/RangeRingUtils.py [hunk 1]
```diff
@@ -106,7 +110,7 @@ class RingMaker:
     def __init__(self, center, inputRangeList, distanceUnits, sr):
         ''' initialize rings '''
 
-        self.deleteme = []
+        self.intermediateDataToDelete = []
```
Comment (csmoore) [inline visibility/toolboxes/scripts/RangeRingUtils.py]: Thank you - this will no longer baffle me every time I see it

**Comment by csmoore** [comment]
Sure - code looks good - I will just need to test the tbxs (assuming you do want me to also test - & they do run/work on 10.3.1 as advertised, since that is what I have)

**Comment by csmoore** [comment]
Testing this now...the unit tests at: https://github.com/mfunk/solutions-geoprocessing-toolbox/tree/dev/utils/test/visibility_tests

ran in ArcMap (10.3.1) but there was a failure in Pro 1.2 - perhaps a little more worrisome is that the unittest assert failure was not picked up by the harness/driver (it reported 0 errors)

```Command
ArcPy works
Scratch already exists
Testing Sun Position Analysis (Pro).
Importing toolbox...
Setting up inputs...
Set date...
Set UTCAfg...
Set output hillshade...
Running tool (Sun Position and Hillshade)
PYTHON ERRORS:
Traceback info:
  File "C:\MyFiles\Work\Github\pr-test\solutions-geoprocessing-toolbox-dev\utils\test\visibility_tests\SunPositionAndHillshadeTestCase.py", line 112, in test_sun_position_analysis
    self.assertEqual(rasMaximum, float(0))

Error Info:
-2.0 != 0.0

ArcPy ERRORS:
Start Time: Tue May 10 09:20:44 2016
Number of unique values = 51
Succeeded at Tue May 10 09:20:44 2016 (Elapsed Time: 0.05 seconds)

RESULTS =================================================

Number of tests run: 13
Number of errors: 0
Number of failures: 0

END OF TEST =========================================
```

**Comment by csmoore** [comment]
A couple of other things I noticed (not really directly related to this PR, but to the unit tests in general):

1. Is it time to remove the old tests - [visibility\test](https://github.com/Esri/solutions-geoprocessing-toolbox/tree/dev/visibility/test) - these confused me a bit (and where I looked first)
2. Adding a script to the path is a little more confusing than it needs to be at:
https://github.com/Esri/solutions-geoprocessing-toolbox/blob/dev/utils/test/visibility_tests/RangeRingUtilsTestCase.py#L44 
Alternate code:

```python
currentPath = os.path.dirname(__file__)
pathToRRUtils= os.path.normpath(os.path.join(currentPath, r"../../../visibility/toolboxes/scripts"))
sys.path.insert(0, pathToRRUtils)   # (could also use append if it didn't need to be first)
import RangeRingUtils
```
(also added [comment here about](https://github.com/mfunk/solutions-geoprocessing-toolbox/commit/a4b60f8fed9756f72234394083785e74eb9b52a6#commitcomment-1741) )

**Comment by mfunk** [comment]
@csmoore since these aren't related to the PR, can you put them into a separate Issue? Thanks.
As for the related ones i'll have to look at it tomorrow or Monday.

**Comment by csmoore** [comment]
Added PR for `#2` (#442)  - `#1` - I assumed is already planned for some point as part of the unit test changes?

**Pull request merged**